code/style.css

```diff
@@ -1,4 +1,478 @@
+  font-weight: bold;
+  font-style: italic;
+  letter-spacing: 1px;
+}
+
+/*** SECTION 4 – footer ***/
+
+.nav-bar-footer {
+  font-size: 16px;
+  margin: 10px;
+  text-transform: uppercase;
+  font-style: italic;
+  color: #000;
+  font-weight: 500;
+}
+
+.social-container {
+  display: block;
+  text-align: center;
+  padding-bottom: 20px;
+  padding-top: 40px;
+}
+
+.social-media {
+  width: 40px;
+  height: 40px;
+  margin: 5px 11px;
+}
+
+.nav-bottom-container {
+  display: flex;
+  justify-content: center;
+  order: 2;
+}
+
+.back-to-top {
+  display: flex;
+  justify-content: center;
+  padding: 12px 0 60px;
+}
+
+.logo-bottom {
+  width: 50px;
+  margin: 0;
+  padding: 10px 0 20px;
+}
+
+.site-name-bottom {
+  display: none;
+}
+
+/****************** MEDIA QUERIES ******************/
+
+/* TABLET */
+@media (min-width: 667px) and (max-width: 1024px) {
+  h2 {
+    margin: 0;
+  }
+
+  /* SECTION 1 - logo & navbar */
+
+  .site-name-top {
+    display: none;
+  }
+
+  .nav-bar {
+    display: flex;
+    flex-direction: row;
+    font-size: 20px;
+    text-transform: uppercase;
+    color: #000;
+    font-weight: 500;
+  }
+
+  .nav-bar:visited {
+    color: #000;
+  }
+
+  .logo-container {
+    order: 2;
+  }
+
+  nav :nth-child(5) {
+    order: 3;
+  }
+  nav :nth-child(6) {
+    order: 3;
+  }
+
+  /* SECTION 2 - the big news content */
+
+  .section-2 {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    padding: 0 20px;
+  }
+
+  .img-container {
+    width: 65%;
+    margin: 0 0 10% 0;
+  }
+
+  .big-news {
+    width: 35%;
+    min-width: 300px;
+  }
+
+  /* SECTION 3 - the other news */
+
+  .section-3 {
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+  }
+
+  .card-container {
+    display: flex;
+    box-sizing: border-box;
+    width: 50%;
+    padding: 0 20px;
+  }
+
+  /* SECTION 4 – footer */
+
+  .section-4 {
+    padding: 40px 20px 20px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    height: 150px;
+  }
+
+  .logo-bottom {
+    display: none;
+  }
+
+  .site-name-bottom {
+    display: flex;
+    text-align: center;
+    font-family: "Lexend Deca", sans-serif;
+    color: #000;
+  }
+
+  .site-name-bottom:visited {
+    color: #000;
+  }
+
+  .social-container {
+    padding-top: 0;
+  }
+
+  .back-to-top {
+    padding: 12px 0 10px;
+    align-self: flex-end;
+  }
+}
+
+/* DESKTOP */
+@media (min-width: 1025px) {
+  body {
+    max-width: 1400px;
+    margin: auto;
+  }
+
+  /* SECTION 1 - logo & navbar */
+
+  .nav-top {
+    justify-content: flex-start;
+  }
+
+  .site-name-top {
+    margin-right: auto;
+    margin-left: 16px;
+    position: relative;
+    animation: mymove 2s ease-in-out;
+  }
+
+  @keyframes mymove {
+    0% {
+      margin-left: 150px;
+      opacity: 0;
+    }
+    50% {
+      opacity: 1;
+      margin-left: 250px;
+    }
+    100% {
+      margin-left: 16px;
+    }
+  }
```

Comment:
Love animation!! If you wanna look into the magical world of animation through code, check out this:
https://animista.net/